src/base64.cr

```diff
@@ -67,6 +67,28 @@ module Base64
     end
   end
 
+  # :nodoc:
+  def encode(data, io : IO)
+    inc = 0
```

Comment:
@kostya This `inc`, `LINE_SIZE` and `NL` logic is duplicated, it also appears below. Could we extract it to a new method that tracks `inc` and yields the appropriate bytes? I don't know the name, probably the current `to_base64` should be `to_strict_base64` and this new one should be `to_base64`, but the names aren't very important. What do you think?